dev-python/spyder-docs/spyder-docs-9999.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit git-r3
+
+DESCRIPTION="Documentation for Spyder"
+HOMEPAGE="https://github.com/spyder-ide/spyder-docs"
+EGIT_REPO_URI="https://github.com/spyder-ide/spyder-docs.git"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS=
+
+BDEPEND="dev-python/sphinx"
```

Comment:
(If you do end upd putting snapshot of this, remember the eclass function for this one)

Reply:
I can't use the distutils functions because `inherit distutils-r1` complains that there is no `setup.py`. How about instead I put the docs part into the spyder ebuild (I will push the changed spyder ebuild once I made the other changes)? Then I can use the `distutils_enable_sphinx` function, and then I don't need a seperate ebuild just for docs.

Reply:
Without reading other comments for now, you can use `DISTUTILS_USE_SETUPTOOLS=no` to bypass that error.